src/app/modules/myinfo/myinfo.service.ts

```diff
@@ -270,6 +272,15 @@ export class MyInfoServiceClass {
       const { fieldValue, isReadOnly } = myInfoData.getFieldValueForAttr(
         myInfoAttr as InternalAttr,
       )
+
+      // Check if field value exists in our constants lists. If it doesn't, log the error
+      if (fieldValue) {
+        const myInfoConstantsList = getMyInfoAttributeConstantsList(myInfoAttr)
+        if (myInfoConstantsList) {
+          isFieldValueInMyinfoList(fieldValue, myInfoAttr, myInfoConstantsList)
```

Comment:
This function actually checks and sends the logger if there is an error right. Perhaps we can make the name more specific?

`logIfFieldValueNotInMyinfoList`

Or at least identify that this method does the logging!